modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/META-INF/mappings/liferay-type-mappings.json

```diff
@@ -1900,6 +1900,16 @@
 			"store": true,
 			"type": "keyword"
 		},
+		"urlSubject": {
+			"fields": {
+				"keyword": {
+					"store": true,
```

Comment:
[MBMessage](https://github.com/liferay/liferay-portal/blob/52fbc1fb4ba7e470f365977c8e56619eb4d87f2d/modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/search/spi/model/index/contributor/MBMessageModelDocumentContributor.java#L104)  looks like the only use of urlSubject, and it's a keyword, so like urlTitle, we should simply make it a keyword mapping with no sub-field.